src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+  display: flex;
+  height: 100vh;
+}
+
+.container {
+  height: 96px;
+  width: 96px;
+  margin: auto;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+}
+
```

Comment:
you will have default selector `.stars__star` that have default
```
  background-image: url(./images/star.svg);
  height: 16px;
  width: 16px;
```
  
and you need to use `:not(:last-child)` to add `margin-right` for all stars besides last one

and you will have something like this t select which stars should have another background
`.stars--1 .stars__star:nth-child(-n + 1)`